Tests/test_PDB_KDTree.py

```diff
@@ -73,7 +73,10 @@ def test_KDTree_exceptions(self):
         coords = random((nr_points, 3)) * 100000000000000
         with self.assertRaises(Exception) as context:
             kdt = kdtrees.KDTree(coords, bucket_size)
-        self.assertTrue("coordinate values should lie between -1e6 and 1e6" in str(context.exception))
+        self.assertTrue(
+            "coordinate values should lie between -1e6 and 1e6"
+            in str(context.exception)
+        )
```

Comment:
Not really black, but I think there is an assert contains method we should be using here.